Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN mkdir -p ${DEPS_HOME}
 WORKDIR ${DEPS_HOME}
 # End
 
-RUN curl -sL https://deb.nodesource.com/setup_14.x | bash - \
+RUN curl -sL https://deb.nodesource.com/setup_16.x | bash - \
```

Comment:
Should it be `setup_18` or are they not tied together?

Reply:
We tried 18 but had some failures in the build, so thought we'd update to 16 for now rather than spend lots of time on it.